backend/open_webui/env.py

```
@@ -349,6 +351,10 @@ def parse_section(section):
     "WEBUI_AUTH_TRUSTED_EMAIL_HEADER", None
 )
 WEBUI_AUTH_TRUSTED_NAME_HEADER = os.environ.get("WEBUI_AUTH_TRUSTED_NAME_HEADER", None)
+WEBUI_AUTH_TRUSTED_GROUPS_HEADER = os.environ.get(
+    "WEBUI_AUTH_TRUSTED_GROUPS_HEADER", None
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)